tests/generic_views/test_dates.py

```diff
@@ -21,10 +21,16 @@ class TestDataMixin:
 
     @classmethod
     def setUpTestData(cls):
+        cls.series1 = Series.objects.create(
+            name='Django books', pubdate=datetime.date(2003, 10, 1)
```

Comment:
Please including a trailing comma in cases like this so if more items are added later, we don't have to modify this line again.